content/features/50-runtimes-monitoring.adoc

```diff
@@ -0,0 +1,15 @@
+---
+title: "Runtimes Monitoring"
+date: 2019-03-06T13:55:38+02:00
+draft: false
+subTitle: "Monitor your application metrics within Kiali"
+image: features/runtimes_monitoring.png
+weight: 50
+---
+
+Kiali comes with some default dashboards for a couple of runtimes: Go, Node.js, Spring Boot, Thorntail and Vert.x.
```

Comment:
minor: "Kiali comes with some default dashboards for a couple of" -> "Kiali comes with default dashboards for several"